comm/README.md

```diff
@@ -55,7 +55,7 @@ The fat jar built above may be run with Java like so:
 
 ```
 $ java -Dfile.encoding=UTF8 -Djava.net.preferIPv4Stack=true -jar target/scala-2.12/src-assembly-0.0.1.jar
-17:34:52.110 [main] INFO main - Listening for traffic on #{Network rnode://ac4b1fdee5a947e8a511383e698df99d@63.224.55.75:30304}.
+17:34:52.110 [main] INFO main - Listening for traffic on #{Network rnode://ace40ebca0924eb797bb69dfda04f5d9@1.2.3.4:30304}.
```

Comment:
Why 1.2.3.4 here but not on line 68?

Reply:
Line 68 is what a docker bridge address typically looks like. 1.2.3.4 is public.